src/store/userProfile/selectors.ts

```diff
@@ -0,0 +1,19 @@
+import { TRootState } from 'store';
+
+import { TUserInfo } from 'components/UserInfo/dictionary';
+
+const selectRootState = (state: TRootState) => state?.user;
```

Comment:
❗️
Обычно рутовые селекторы как-то так называются. Плюс можно посмотреть в сторону [reselect](https://github.com/reduxjs/reselect)
```suggestion
const userStateSelector = (state: TRootState) => state?.user;
```